src/MassTransit.Abstractions/SagaStateMachine/Contracts/RequestTimeoutExpired.cs

```diff
@@ -25,5 +25,10 @@ public interface RequestTimeoutExpired<TRequest>
         /// The requestId of the request
         /// </summary>
         Guid RequestId { get; }
+
+        /// <summary>
+        /// The  original request message. 
+        /// </summary>
+        TRequest Message { get; }
```

Comment:
Should probably still be a nullable property, since there is no guarantee it's there due to version changes.